scripts/docs/generateJsonSchemas.ts

```diff
@@ -2,6 +2,8 @@ import * as fs from 'fs/promises';
 import * as path from 'path';
 import { zodToJsonSchema } from 'zod-to-json-schema';
 import * as z from 'zod';
+// TODO: Enable this rule, make eslint recognize the import path
+// eslint-disable-next-line import/no-relative-packages
 import { META } from '../../packages/toolpad-studio/src/server/schema';
```

Comment:
Couldn't solve this one, I think it's because `@toolpad/studio` only exports certain things - didn't want to mess with that just for this eslint rule.